ansible_collections/arista/avd/roles/eos_designs/docs/how-to/avd-to-cvaas.md

```diff
@@ -42,6 +45,7 @@ ansible_httpapi_port: 443
 ```
```

Comment:
instea of having the step in the heading we could have a simplified heading title and the instructions after

```
## Ading group variables for the CVaaS instance

Create a folder under `group_vars` named `CVAAS` and add a file named `cvaas_auth.yml`. Your file should look similar to this:```

Reply:
Sounds good, updated.